_episodes/03-matplotlib.md

```diff
@@ -130,10 +130,24 @@ formats, including SVG, PDF, and JPEG.
 
 > ## Importing libraries with shortcuts
 >
-> When importing the `pyplot` module of `matplotlib` it is common to use the shortcut
-> `import matplotlib.pyplot as plt`. If we do this we then must write `plt.plot(...)`
-> instead of `matplotlib.pyplot.plot(...)`, for example. This is popular as it saves
-> a lot of typing!
+> In this lesson we use the `import matplotlib.pyplot` [syntax]({{ page.root }}/reference.html#syntax)
+> to import the `pyplot` module of `matplotlib`. However, shortcuts such as
+> `import matplotlib.pyplot as plt` are frequently used.
+> Importing `pyplot` this way means that after the initial import, rather than writing
+> `matplotlib.pyplot.plot(...)`, you can now write `plt.plot(...)`.
+> Another common convention to use the shortcut `import numpy as np` when importing the
```

Comment:
I think this is missing a word...perhaps it should be "Another common convention **is**"...

Reply:
Thanks, just pushed a commit to add the "is".